File: core/core/src/main/java/cn/netdiscovery/core/domain/Response.java
package cn.netdiscovery.core.domain;

import lombok.Data;

import java.io.InputStream;
import java.io.Serializable;

/**
 * Created by tony on 2018/1/20.
 */
@Data
public class Response implements Serializable {

    private static final long serialVersionUID = 7518499261167741602L;

    private byte[] content;
    private int statusCode;
    private String contentType;
    private InputStream is;
}

File: core/core/src/main/kotlin/cn/netdiscovery/core/domain/Response.kt
package cn.netdiscovery.core.domain

import java.io.InputStream
import java.io.Serializable

/**
 *
 * @FileName:
 *          cn.netdiscovery.core.domain.Response
 * @author: Tony Shen
 * @date: 2020-10-10 16:06
 * @version: V1.0 对各个网络框架返回的 response 的封装
 */
class Response : Serializable {

    var content: ByteArray?=null
    var statusCode:Int = 0
    var contentType: String? = null
    var `is`: InputStream? = null
}